src/test/java/com/booleanuk/core/BasketTest.java

```diff
@@ -3,6 +3,134 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+import java.util.ArrayList;
+
 class BasketTest {
 
+    //User Story 1
+    @Test
+    public void bagelAddedToBasketReturnTrue() {
+        Basket basket = new Basket();
+
+        basket.addBagelTypeToBasket("Chocolate Bagels");
+
+        boolean bagelAdded = basket.bagels.contains("Chocolate Bagels");
+
+        Assertions.assertTrue(bagelAdded);
```

Comment:
I would probably add another test that also made sure I could not add another item if that item is already in the basket.